crates/router/src/compatibility/stripe/payment_intents/types.rs

```diff
@@ -171,11 +237,17 @@ pub struct StripePaymentIntentRequest {
     pub client_secret: Option<pii::Secret<String>>,
     pub payment_method_options: Option<StripePaymentMethodOptions>,
     pub merchant_connector_details: Option<admin::MerchantConnectorDetailsWrap>,
-    pub mandate_id: Option<String>,
+    pub mandate: Option<String>,
     pub off_session: Option<bool>,
-    pub payment_method_type: Option<api_enums::PaymentMethodType>,
+    pub payment_method_types: Option<api_enums::PaymentMethodType>,
     pub receipt_ipaddress: Option<String>,
     pub user_agent: Option<String>,
+    pub mandate_data: Option<MandateData>,
+    pub automatic_payment_methods: Option<secret::SecretSerdeValue>, // not used
+    pub payment_method: Option<String>, // not used
+    pub confirmation_method: Option<String>, //not used
+    pub error_on_requires_action: Option<String>, // not used
+    pub radar_options: Option<secret::SecretSerdeValue> // not used
```

Comment:
Lets say the merchant is passing `radar_options` or `error_on_requires_action` and he is expecting some action based on that. If hyperswitch doesn't support that feature, it is better to throw error saying the feature is not supported. 

But if there is no action expected based on the field passed. Then maybe we can accept them without throwing an error.

Reply:
We don't have functionality for these fields. So shall we just take them and do nothing? Or shall we throw an error saying that this feature is not supported?